From e99828ec6a2fd815f0b6c907882ea9932661a8c8 Mon Sep 17 00:00:00 2001
From: Kris Bahnsen <kris@embeddedTS.com>
Date: Mon, 2 May 2022 16:26:38 -0700
Subject: [PATCH] scripts: Adjust TS-SILO script name and install

The current ts7680-utils repo is based around it specifically
using Debian with systemd. Thus, the Makefile installs systemd
scripts that should not go here in buildroot.

Additionally, the TS-SILO supercapacitor script in ts7680-utils
is named differently than all of the subsequent platforms that
also have these supercapacitors. Rename it to "tssilomon" so that
our supporting Buildroot init package is compatible

Signed-off-by: Kris Bahnsen <kris@embeddedTS.com>
---
 scripts/Makefile.am                  | 10 +---------
 scripts/{tssupercapmon => tssilomon} |  0
 2 files changed, 1 insertion(+), 9 deletions(-)
 rename scripts/{tssupercapmon => tssilomon} (100%)

diff --git a/scripts/Makefile.am b/scripts/Makefile.am
index abb5763..39fd129 100755
--- a/scripts/Makefile.am
+++ b/scripts/Makefile.am
@@ -1,9 +1 @@
-bin_SCRIPTS = tssupercapmon tsincmac tsstartupscript
-
-# Note: This is a horrible abuse of autotools, but these services need to go to
-# the /etc path to match existing setup
-systemdservicedir = /etc/systemd/system/
-systemdservice_SCRIPTS = tssetmac@.service tsstartup.service tssupercapmon.service tsswitch.service
-
-systemdnetdir = /etc/systemd/network/
-systemdnet_SCRIPTS = eth0.1.link eth0.1.netdev eth0.1.network eth0.2.link eth0.2.netdev eth0.2.network switch.network vlan.network
+bin_SCRIPTS = tssilomon
diff --git a/scripts/tssupercapmon b/scripts/tssilomon
similarity index 100%
rename from scripts/tssupercapmon
rename to scripts/tssilomon
-- 
2.11.0

